python/k8s.py

```diff
@@ -488,28 +488,80 @@ def install_ingress_nginx(version):
 
 
 @k8s.command()
-@option('--version', default='v14.6.0', help="The version of prometheus chart to install")
+@option('--version', default='v18.0.2', help="The version of kube-prometheus-stack chart to install")
 @option('--alertmanager', default=False, help="Enable alertmanager")
 @option('--pushgateway', default=False, help="Enable pushgateway")
-@option('--retention', default='1d', help="Server retention")
-@option('--persistent-volume-size', default='1Gi', help="Server persistent volume size")
-def install_prometheus(version, alertmanager, pushgateway, retention, persistent_volume_size):
-    """Install a prometheus instance in the current cluster"""
+@option('--coredns', default=False, help="Enable coreDns")
```

Comment:
not strictly required, but we usually give the user a chance to deactivate an option with `'--coredns/--no-coredns'`

Reply:
that way it can be deactivated even when the activation is recorded in the parameters